app/reducers/index.js

```diff
@@ -229,3 +183,93 @@ export const surveySubmissionSchema = new schema.Entity('surveySubmissions', {
   user: userSchema,
 });
 export const tagSchema = new schema.Entity('tags', {}, { idAttribute: 'tag' });
+
+const followersSchemaGenerator = (
+  title,
+  types: {
+    fetch?: EntityReducerTypes,
+    mutate?: EntityReducerTypes,
+    delete?: EntityReducerTypes,
+  } = {}
+) => {
+  const key = `followers${title}`;
+  return [
+    new schema.Entity(key, {
+      follower: userSchema,
+    }),
+    createEntityReducer({
+      key,
+      types,
+    }),
+  ];
+};
+
+export const [followersEventSchema, followersEvent] = followersSchemaGenerator(
+  'Event',
+  {
+    mutate: [Event.FOLLOW],
+    delete: [Event.UNFOLLOW],
+  }
+);
+
+export const [
+  followersCompanySchema,
+  followersCompany,
+] = followersSchemaGenerator('Company');
+
+export const [followersUserSchema, followersUser] = followersSchemaGenerator(
+  'User'
+);
```

Comment:
Maybe a bit random to have this here? I think it's a bit cleaner to have this with the followers selectors.

Reply:
Guess we can do that, but then we end with twice as much code, and need to defined the various following types twice (bigger change for errors), but guess we can move them.